uxdot/uxdot-demo.ts

```diff
@@ -48,7 +48,7 @@ export class UxdotDemo extends LitElement {
                      icon-set="ui"
                      @click="${this.#reloadIframe}">Reload</rh-button>
           <rh-cta slot="footer" href="${this.demoSourceUrl}">View source on GitHub</rh-cta>
-          <rh-cta slot="footer" href="${this.demoUrl}">View In Own Tab</rh-cta>
+          <rh-cta slot="footer" href="${this.demoUrl}" target="_blank" rel="noreferrer noopener">View In Own Tab</rh-cta>
```

Comment:
Thanks for tackling this @rohit-bharmal! It was one of the things on my to-do list.

I could be wrong, but I don't believe we need the `rel="noreferrer noopener"` on these demo links since they're our own, internal links.

Reply:
Makes sense to me